projects/laji-api-client-b/README.md

```diff
@@ -0,0 +1,20 @@
+Updating api type definitions:
+
+```
+npm i
+
+// update types based on the new API openapi schema
+npm run updateNew
+```
+
+Usage:
+```typescript
+export class ExampleComponent {
+  constructor(private lajiApiClientService: LajiApiClientBService) {}
+  function f() {
+    const a$ = this.lajiApiClientService.fetch('/collections', 'get', { query: { page: 1 } });
+  }
+}
+```
+
+Queries are cached by default based on the path and the query params. Whenever a new non-get request is sent to the same path, the cache is flushed automatically.
```

Comment:
This is out of date. If it's hard to document, maybe put a link to the review discussion that explains it? Or just remove it?

Reply:
Or idk... it's not entirely out of date. It says "based on the path", which is correct. I mean it doesn't mention that the path params affect it also. Maybe we can update the README after we develop the other changes (like method wrappers `get()` `put()` etc)